specification/dimensions/billingperiodstart.md

```diff
@@ -0,0 +1,32 @@
+# Billing Period Start
+
+Billing period represents the time window for which you have or will receive an invoice for. Billing period represents the time window for which you have or will receive an invoice for. The time window is inclusive of the start date and exclusive of the end date.
+
+Billing Period Start represents the beginning of a billing period.
```

Comment:
Should we be explicit about it being a date and time? I think we should use "the" since this applies to the value for the row.
```suggestion
Billing Period Start represents the start date and time of the billing period.
```